### PR TITLE
enh(documentation): Improve installation chapters

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [x] 22.04.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 22.10.x (next)

### CODE REVIEW

**Review by smau-centreon — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-poller/using-packages.md [hunk 1]
```diff
@@ -338,11 +402,11 @@ Failed connect to 192.168.0.1:444; Connection refused
 
 > La version Centreon du serveur distant est invalide. Elle doit être supérieure ou égale à 22.04.
 
-## Ajouter le Poller à la configuration
+## Étape 4 : Ajouter le Poller à la configuration
 
 Rendez-vous au chapitre [Ajouter un Poller à la configuration](../../monitoring/monitoring-servers/add-a-poller-to-configuration.md).
 
-## Sécurisez votre plateforme
+## Étape 5 : Sécurisez votre plateforme
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-poller/using-packages.md:409]:
```suggestion
## Étape 5 : Sécuriser votre plateforme
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by smau-centreon — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-central-server/using-packages.md [hunk 2]
```diff
@@ -401,9 +443,9 @@ Sécurisez l'accès root à MariaDB en exécutant la commande suivante :
 mysql_secure_installation
 ```
 
-> Vous devez obligatoirement définir un mot de passe pour l'utilisateur root de la base de données.
+> Définissez un mot de passe pour l'utilisateur root de la base de données.
 
-Enfin, dans la base de données distante, créez un utilisateur avec privilèges **root**. Vous devrez renseigner cet utilisateur pendant le processus d'installation web (à [l'étape 6](../web-and-post-installation.md#étape-6--database-information), dans les champs **Root user** et **Root password**).
+Enfin, dans la base de données distante, créer un utilisateur avec privilèges **root**. Renseignez cet utilisateur pendant le processus d'installation web (à [l'étape 6](../web-and-post-installation.md#étape-6--database-information), dans les champs **Root user** et **Root password**).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-central-server/using-packages.md:448]:
```suggestion
Enfin, dans la base de données distante, créez un utilisateur avec privilèges **root**. Renseignez cet utilisateur pendant le processus d'installation web (à [l'étape 6](../web-and-post-installation.md#étape-6--database-information), dans les champs **Root user** et **Root password**).
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-central-server/using-packages.md [hunk 3]
```diff
@@ -113,7 +150,8 @@ dnf install -y https://rpms.remirepo.net/enterprise/remi-release-8.rpm
 dnf config-manager --set-enabled 'powertools'
 ```
 
-Activez PHP 8.0 en utilisant les commandes suivantes :
+Activer PHP 8.0 en utilisant les commandes suivantes :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-central-server/using-packages.md:153]:
```suggestion
Activez PHP 8.0 en utilisant les commandes suivantes :
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-central-server/using-packages.md [hunk 4]
```diff
@@ -525,7 +565,7 @@ La timezone par défaut de PHP doit être configurée.
 <Tabs groupId="sync">
 <TabItem value="Alma / RHEL / Oracle Linux 8 / CentOS 7" label="Alma / RHEL / Oracle Linux 8 / CentOS 7">
 
-Exécutez la commande suivante en `root` :
+Exécutez la commande suivante :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-central-server/using-packages.md:568]: Why not as root?

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-remote-server/using-packages.md [hunk 5]
```diff
@@ -37,28 +57,40 @@ apt update && apt upgrade
...
+## Étape 1 : Pré-installation
+
+### Désactiver SELinux
 
-Si vous installez Centreon sur AlmaLinux/RHEL/OracleLinux 8, et que vous comptez utiliser Centreon en français, espagnol ou portugais, installez les paquets correspondants :
+<Tabs groupId="sync">
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
+
+Pendant l'installation, SELinux doit être désactivé. Éditez le fichier
+**/etc/selinux/config** et remplacer **enforcing** par **disabled**, ou bien
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-remote-server/using-packages.md:71]:
```suggestion
**/etc/selinux/config** et remplacez **enforcing** par **disabled**, ou bien
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-remote-server/using-packages.md [hunk 6]
```diff
@@ -567,22 +605,21 @@ systemctl restart mariadb
 
 ### Sécuriser la base de données
 
-Depuis MariaDB 10.5, il est obligatoire de sécuriser l'accès en root à la base avant d'installer Centreon. Si vous utilisez une base de données locale, exécutez la commande suivante sur le serveur central :
+Depuis MariaDB 10.5, il est obligatoire de sécuriser l'accès en root à la base avant d'installer Centreon. Si vous utilisez une base de données locale, exécuter la commande suivante sur le serveur central :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-remote-server/using-packages.md:608]:
```suggestion
Depuis MariaDB 10.5, il est obligatoire de sécuriser l'accès en root à la base avant d'installer Centreon. Si vous utilisez une base de données locale, exécutez la commande suivante sur le serveur central :
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by thiuyendang — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Pull request merged**